app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <activity
+            android:name=".sample.CameraCaptureActivity"
+            android:screenOrientation="portrait" />
```

Comment:
Will it conflict with #16 ? 🤔 
BTW, Could you please add `tools:ignore="LockedOrientationActivity"` same as [here](https://github.com/nimblehq/alpine/pull/16/files#diff-7fa6aef292187a049f7a4d6060d8df3ba212d838789c78940bd363344b1c38cdR30)?

Reply:
Hmm 🤔 let me check

Reply:
When merging, maybe there are some conflicts because we were editing the same file `AndroidManifest.xml` but it is easy to resolve so I think it does not matter much 😆 Thanks for letting me know anw @lydiasama